sdk/tables/ci.yml

```diff
@@ -25,6 +25,7 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: tables
+    BuildSnippets: false
```

Comment:
You can revert this - I fixed the snippets in https://github.com/Azure/azure-sdk-for-net/pull/33998

Reply:
As long as the validation on this PR takes, I'd rather wait till after so it just touches the _sdk/tables_ directory. I'll leave this unresolved for now so I don't forget while the 1 failed run on MacOS re-runs.